packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -218,6 +218,7 @@ export default class TradeStore extends BaseStore {
         reaction(
             () => [this.contract_type],
             () => {
+                this.root_store.portfolio.setContractType(this.contract_type);
```

Comment:
we need to update the contract_type for portfolio store since we are using this state in the contract-trade store(in the core package) and open-positions.jsx( in the reports).